person_capture/gui_app.py

```
@@ -14356,16 +14369,16 @@ def _hdr_tonemap_filter_cmds(
-        if algo not in {"auto", "spline", "bt.2390", "st2094-40", "mobius", "hable", "reinhard", "clip"}:
-            algo = "bt.2390"
+        algo = str(getattr(self.cfg, "hdr_sdr_tonemap", "spline") or "spline").lower()
+        if algo not in {"auto", "spline", "bt.2390", "bt2390", "st2094-40", "mobius", "hable", "reinhard", "clip"}:
+            algo = "spline"
         gamut = str(getattr(self.cfg, "hdr_sdr_gamut_mapping", "perceptual") or "perceptual").lower()
         if gamut not in {"perceptual", "relative", "saturation", "clip", "desaturate", "darken", "linear"}:
             gamut = "perceptual"
         try:
-            contrast_recovery = max(0.0, min(2.0, float(getattr(self.cfg, "hdr_sdr_contrast_recovery", 0.30))))
+            contrast_recovery = max(0.0, min(2.0, float(getattr(self.cfg, "hdr_sdr_contrast_recovery", 0.0))))
         except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)